packages/playground/src/components/pull-request/pull-request-list.tsx

```diff
@@ -211,26 +211,28 @@ export default function PullRequestList() {
       <div className="bg-background border-b border">
         <div className="flex p-3">
           <div
-            className={`flex justify-center text-center ${headerFilter === 'open' ? 'text-white' : ''}`}
+            className={`flex justify-center text-center ${headerFilter === 'open' ? 'text-white' : 'text-[#93939F]'}`}
```

Comment:
let's not hardcode colors.. add correct tailwind classes.. and use the `classnames` or `clsx` package to apply classes conditionally. 
avoid using string literals for that.

Reply:
`className={cx("flex justify-center text-center", 
{'text-white' : headerFilter === 'open' }, 
{'text-[#93939F]' : headerFilter !== 'open' })}`

Reply:
done